MANIFEST.in

```diff
@@ -14,4 +14,4 @@ include README.md
 include CHANGELOG.md
 
 include jupyterbook_latex/py.typed
-recursive-include jupyterbook_latex/theme *
+include jupyterbook_latex/theme/jupyterBook.cls
```

Comment:
@chrisjsewell just wanted to know the thinking behind this. in some places we use * like https://github.com/executablebooks/jupyter-book/blob/master/MANIFEST.in#L29, and in some don't. My thinking was that in the future we might have some `.sty` files and other theme-specific files in this folder, and the * declaration will make it future-proof.

Reply:
The `check-manifest` pre-commit hook already keeps it future proof

Reply:
awesome.